HW-22/database22(2).py

```diff
@@ -0,0 +1 @@
+# SELECT users.id, users.first_name, users.last_name, books.title FROM purchases p, users JOIN  books ON users.id = p.user_id and p.book_id = books.id ORDER BY users.id
```

Comment:
When you select from multiple tables `FROM purchases p, users` it means that you join these tables using full join. After that you will have a table with `table1.len X table2.len` which is dangerous when you work with big amount of data. 

Use multiple join instead: `... JOIN books on users.id = p.users_id join books on books.id = p.book_id ... `